sdk/identity/azure-identity/TOKEN_CACHING.md

```diff
@@ -34,6 +36,12 @@ With persistent disk token caching enabled, the library first determines if a va
 - Ensures the app can continue to function during an Azure AD outage or disruption.
 - Avoids having to prompt users to authenticate each time the process is restarted.
 
+#### Implementation Note
+
+The cache is stored in two files to allow for separate caching of Continuous Access Evaluation (CAE) vs non-CAE credentials. The default filename is `msal` but can be specified through `TokenCachePersistenceOptions.setName()`. The filenames end in `.cae` or `.nocae` respectively.
```

Comment:
And this isn't the whole story because we cache in files only on Windows and when falling back to plaintext on other platforms. I think we should just leave all this as undocumented implementation details i.e., delete this section. What should customers do with this information whose continued correctness we won't guarantee?

Reply:
Yeah, I'm torn on this. There was some discussion in one of our meetings about this around customers who might have somehow taken a dependency on the existing file. I am fine to remove it from here. @joshfree, I think this was one you had opinions about. (At least you were the last one to say something before I wrote it down.)

Reply:
Removed the section after discussion.